src/handbook/company/security/incident-response.md

```diff
@@ -0,0 +1,163 @@
+and events, and offers guidance for employees or incident responders who believe
+they have discovered, or are responding to, a security incident.
+
+## Scope 
+
+This policy covers all information security or data privacy events or incidents.
+
+## Incident and Event Definitions
+
+A **security event** is an observable occurrence relevant to the confidentiality,
+availability, integrity, or privacy of company controlled data, systems or networks.
+
+A **security incident** is a *security event* which results in loss or damage to
+the confidentiality, availability, integrity, or privacy of company controlled
+data, systems or networks.
+
+## Security Incident Response Team (SIRT)
+
+The Security Incident Response Team (SIRT) is responsible for
+
+ - Reviewing analyzing, and logging all received reports and tracking their statuses.
+ - Performing investigations, creating and executing action plans, and post-incident activities.
+ - Collaboration with law enforcement agencies.
+
+Current members of the FlowForge SIRT:
+
+ - CTO
+ - CEO
+ - Customer Success manager
+ - Product Owner
+
+
+## Incident Reporting & Documentation
+
+### Reporting 
+
+If a FlowForge employee, contractor, user, or customer becomes aware of an
+information security event or incident, possible incident, imminent incident,
+unauthorized access, policy violation, security weakness, or suspicious activity,
+then they shall immediately report the information using one of the following
+communication channels:
+
+ - Email security@flowforge.com with information or reports about the event or
+   incident
+ - Raise an issue in [`flowforge/security`](https://github.com/flowforge/security){rel="nofollow"}
+ - Report it in #security channel in slack
+
+Reports should include specific details about what has been observed or discovered.
+
+
+### Severity 
+
+The [SIRT](#security-incident-response-team-sirt) shall monitor incident and event
+tickets and shall assign a ticket severity based on the following categories:
+
+
+ - **P3/P4** - Low and Medium Severity
+    Issues meeting this severity are simply suspicions or odd behaviors. They
+    are not verified and require further investigation. There is no clear indicator
+    that systems have tangible risk and do not require emergency response. This
+    includes lost/stolen laptop with disk encryption, suspicious emails, outages,
+    strange activity on a laptop, etc.
+ - **P2** - High Severity
+    High severity issues relate to problems where an adversary or active
+    exploitation hasn't been proven yet, and may not have happened, but is likely
+    to happen. This may include lost/stolen laptop without encryption, vulnerabilities
+    with direct risk of exploitation, threats with risk or adversarial persistence
+    on our systems (e.g.: backdoors, malware), malicious access of business
+    data (e.g.: passwords, vulnerability data, payments information).
+ - **P1** - Critical Severity
```

Comment:
Do we have these as labels on GH? It would be great if you add a process with the policy.

Reply:
Yes - these are the labels we have on the security repo. I'll try and pull it together with https://flowforge.com/handbook/development/security a bit more.

Reply:
Have made it more explicit that these are assigned via labels on the GH issue - swapping the 'ticket' wording for 'issue' as well to be clearer.